messages/scheduler/scheduler.bg-BG.yml

```diff
@@ -324,3 +324,48 @@ kendo:
 
     # The value of the aria-label attribute of the <select> element used for switching the Scheduler views.
     selectView: Избери изглед
+
+    # The 'Today' button text in the popup of the DateTimePickers and DatePickers in the scheduler event editor.
```

Comment:
Let's make "scheduler" start with an uppercase letter -> "Scheduler". Should be applied everywhere.